packages/shared/ReactFeatureFlags.js

```diff
@@ -123,3 +123,6 @@ export const enableLegacyFBSupport = false;
 // interleaved event. Remove this flag once we have migrated to the
 // new behavior.
 export const deferRenderPhaseUpdateToNextBatch = true;
+
+// Replacement for runWithPriority in React internals.
+export const enableCurrentUpdateLanePriority = true;
```

Comment:
TODO: disable this before landing, enabling it to see the test failures in CI

Reply:
You can run `yarn test-www-variant` to run the tests with the flag on.

Reply:
(We have too many test commands and combinations, sorry it's so confusing. I want to replace it with a single command with multiple arguments, like `yarn test --variant --release_channel=FB_WWW`.)